Mapping/inc/Obstacles.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Header file providing class for obstacles
+ * Author: Frederick Wachter
+ */
+
+#ifndef __OBSTACLES__
+#define __OBSTACLES__
+
+#include <array>
+using namespace std;
+
+class Obstacles {
+	public:
+		bool *directions[4] = { &north, &east, &south, &west };
```

Comment:
How do I do bracket initialization of an array that is setup the way I have it when the initialization is not being done when the variable is being declared?